WikiDrips/WikiRequest.swift

```diff
@@ -17,9 +17,9 @@ public class WikiRequest
         self.searchText = searchText
     }
     
-    public func fetchWikiDocs(handler: @escaping ([WikiDoc]) -> Void) {
+    public func fetchWikiDocs(handler: @escaping ([WikiDoc]) -> Void) -> URLSessionDataTask? {
```

Comment:
Ok, I've made the name change.  But I'm also thinking I should try to follow the pattern.  So I'll try that in a separate pr for handling errors.